src/applications/covid-research-volunteer/containers/ConfirmationPage.jsx

```diff
@@ -55,6 +57,9 @@ export class ConfirmationPage extends React.Component {
             We won’t contact you. But we’ll keep your information on file for
             future studies you may be eligible to join.
           </p>
+          <a href="preview-prod.vfs.va.gov/preview?nodeId=6785">
```

Comment:
is this supposed to be the same as the other link?

Reply:
Need to confirm these links with content folks

Reply:
These links are supposed to be the same, currently planned for `/coronavirus-research`. updated locally and will push up.